src/components/Top-Bar.vue

```diff
@@ -7,7 +7,7 @@
     </oc-topbar-item>
 
     <oc-topbar-item slot="right">
-      <oc-icon name="account_circle" color="white"></oc-icon>
+      <ocAvatar class="uk-margin-small-right" :userid="user.displayname" />
```

Comment:
@DeepDiver1975 from what I found, we dont have the userid in the user object here - thoughts?

Reply:
should be user.id

Reply:
ah - missing in vuex

![Screenshot from 2019-03-28 11-52-25](https://user-images.githubusercontent.com/1005065/55152136-03882b80-5150-11e9-970a-2455cc3230e5.png)

Reply:
pushed a commit to add the is - now having troubles with the initial binding - see my comment on watching the prop .....